### PR TITLE
[#389] Update Bootstrap 5.1.3 to 5.2.3 to resolve deprecated issues

### LINKED ISSUE #389: Update Bootstrap 5.1.3 to 5.2.3 to resolve deprecated issues
## Why

In order to fix some deprecation warnings, we updated Bootstrap in our client project:
image

This requires also adding 1 new SCSS import (`maps.scss`) and allow to benefit from some other minor fixes/changes.
See more at: https://getbootstrap.com/docs/5.2/migration/#v520
 
## Who Benefits?

Developers starting a new project: No need to manually update Bootstrap during the project start.

### PR DESCRIPTION
**Pull request by malparty**
- close #389

## What happened 👀

Update bootstrap 5.1.3 to 5.2.3

## Insight 📝

Added import guidances in comments following the [Bootstrap documentation](https://getbootstrap.com/docs/5.2/migration/#new-_mapsscss).

image

### CSS can build:

image

### Project run and can refer to bootstrap classes:

_Example provided with margins, form-control and buttons_

image

### CODE REVIEW

**Review by andyduong1920 — APPROVED** [review]

File: .template/addons/bootstrap/stylesheets/vendor/_bootstrap.scss [hunk 1]
```diff
@@ -5,7 +5,10 @@
 
 // Configuration
 @import 'bootstrap/scss/functions';
+// @import 'variables'; // Optional variable overrides here
```
Comment (olivierobert) [inline .template/addons/bootstrap/stylesheets/vendor/_bootstrap.scss:8]: Now, this file is imported in the manifest file, so I am curious about how the `application.scss` would look like:

image

https://github.com/onward-venture/ewa-payroll-web/blob/c520ab422a4a24c6a2fb31d6aca50bf8ec27c548/app/assets/stylesheets/_core.scss#L2-L8

Comment (malparty) [reply .template/addons/bootstrap/stylesheets/vendor/_bootstrap.scss:8]: Nice catch, I missed this one!

Comment (malparty) [reply .template/addons/bootstrap/stylesheets/vendor/_bootstrap.scss:8]: Let me remove this line at the moment, as at the moment we always use `variables` without the `bootstrap/scss/functions` (the reason why we can't use `rem()` function in the variables. 

I will open a new issue for this case, as I believe it is a nice improvement but I need to research more about how to have a clean import (I'm thinking having 2 vendors import in application.scss in order to keep the `variables` import outside from the `vendro` folder 👍 )

File: .template/addons/bootstrap/stylesheets/vendor/_bootstrap.scss [hunk 2]
```diff
@@ -5,7 +5,10 @@
...
 @import 'bootstrap/scss/variables';
+@import 'bootstrap/scss/maps';
+// @import 'sass_maps'; // Optional Sass map overrides here
```
[the remark this answers is not in the document]
Comment (malparty) [reply .template/addons/bootstrap/stylesheets/vendor/_bootstrap.scss:10]: 3dca276 ✅

File: .template/addons/bootstrap/stylesheets/vendor/_bootstrap.scss (hunk 1 above)
Comment (malparty) [reply .template/addons/bootstrap/stylesheets/vendor/_bootstrap.scss]: 3dca276

Comment (malparty) [reply .template/addons/bootstrap/stylesheets/vendor/_bootstrap.scss]: ... and issue created in https://github.com/nimblehq/rails-templates/issues/393

**Review by olivierobert — APPROVED** [review]

**Pull request merged**